pylib/gyp/input.py

```diff
@@ -1619,10 +1619,10 @@ def ExpandWildcardDependencies(targets, data):
                 index = index + 1
 
 
-def Unify(l):
-    """Removes duplicate elements from l, keeping the first element."""
+def Unify(list):
```

Comment:
`list` is the name of a Python data type so it should never be used as a variable name.